tests/fixtures/example.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
this file should probably be trimmed down into few lines. 
and with random data :)

Reply:
Can you show the sample `json` file with random data to make sure your meaning?